data/json/items/electronics.json

```diff
@@ -48,5 +48,18 @@
     "material": [ "copper" ],
     "symbol": "|",
     "color": "brown"
+  },
+  {
+    "type": "GENERIC",
+    "id": "melchior_platter",
+    "name": { "str": "supercomputer platter" },
+    "description": "A hefty and expensive-looking golden dish engraved with concentric circuitry.  You felt the urge to rip it from a supercomputer a while ago.",
```

Comment:
Isn't it weird to reference "a while ago" when this will be immediatly visible as soon as the item spawns? Or is it intended time shenanigans?